content/docs/1.4.0/deploy/install/_index.md

```diff
@@ -182,19 +182,30 @@ iscsi install successfully
 
 ### Installing NFSv4 client
 
-The command used to install a NFSv4 client differs depending on the Linux distribution.
+In Longhorn system, backup feature requires NFSv4, v4.1 and v4.2, and ReadWriteMany (RWX) volume feature requires NFSv4.1. Before installing NFSv4 client userspace daemon and utilities, make sure the client kernel support is enabled on each Longhorn node.
 
-For Debian and Ubuntu, use this command:
+- Check `NFSv4.1` support is enabled in kernel
+  ```
+  cat /boot/config-`uname -r`| grep CONFIG_NFS_V4_1
+  ```
 
-```
-apt-get install nfs-common
-```
+- Check `NFSv4.2` support is enabled in kernel
+  ```
+  cat /boot/config-`uname -r`| grep CONFIG_NFS_V4_2
+  ```
 
-For RHEL, CentOS, and EKS with `EKS Kubernetes Worker AMI with AmazonLinux2 image`, use this command:
 
-```
-yum install nfs-utils
-```
+The command used to install a NFSv4 client differs depending on the Linux distribution.
+
+- For Debian and Ubuntu, use this command:
+  ```
+  apt-get install nfs-common
+  ```
+
+- For RHEL, CentOS, and EKS with `EKS Kubernetes Worker AMI with AmazonLinux2 image`, use this command:
```

Comment:
Let's add a command for openSUSE/SLES as well.

Reply:
Updated. Thank you.